packages/@dcl/react-ecs/src/reconciler/index.ts

```diff
@@ -116,18 +96,23 @@ export function createReconciler(
         update.props as EventSystemCallback,
         {
           button: InputAction.IA_POINTER,
-          hoverText: ''
+          hoverText: '@dcl/react-ecs/onPointerDown'
```

Comment:
is this string necessary?

Reply:
yep, this creates the PointerEvent component in the background (PointerHoverFeedback).

Reply:
why are we serializing and sending that string though?

Reply:
https://github.com/decentraland/js-sdk-toolchain/pull/357/commits/addd2f1d416bf0556909e57f53bb0d7f97d01e7b#diff-cb93e4e4437dc40ff0f226b8a5587deaa008fe1926da8b250fec3935ceaf0a53R108-R110